docs/porting/security/psa_targets_in_mbed-os.md

```diff
@@ -0,0 +1,561 @@
+
+As explained above, dual-core and Armv8-M PSA targets require only an NSPE target name. For Armv8-M non-PSA targets, you can define both SPE and NSPE target names.
+
+The naming convention is:
+
+- `TargetName`: PSA non-secure target (NSPE).
+- `TargetName_NPSA_S`: Non-PSA secure target.
+- `TargetName_NPSA`: Non-PSA non-secure target.
+
+### Adding single-core PSA targets
+
+Mbed OS's PSA service emulation provides PSA compatibility for single-core PSA
+targets.
+
+The following example shows a PSA-enabled single-core target, `K64F`:
+
+```json
+    "K64F": {
+        "supported_form_factors": [
+            "ARDUINO"
+        ],
+        "components_add": [
+            "SD",
+            "FLASHIAP"
+        ],
+        "core": "Cortex-M4F",
+        "supported_toolchains": [
+            "ARM",
+            "GCC_ARM",
+            "IAR"
+        ],
+        "extra_labels_add": [
+            "Freescale",
+            "MCUXpresso_MCUS",
+            "KSDK2_MCUS",
+            "FRDM",
+            "KPSDK_MCUS",
+            "KPSDK_CODE",
+            "MCU_K64F",
+            "Freescale_EMAC"
+        ],
+        "is_disk_virtual": true,
+        "macros_add": [
+            "CPU_MK64FN1M0VMD12",
+            "FSL_RTOS_MBED",
+            "MBED_SPLIT_HEAP",
+            "MBED_TICKLESS"
+        ],
+        "inherits": [
+            "PSA_V7_M"
+        ],
+        "detect_code": [
+            "0240"
+        ],
+        "device_has_add": [
+            "USTICKER",
+            "LPTICKER",
+            "RTC",
+            "CRC",
+            "ANALOGIN",
+            "ANALOGOUT",
+            "EMAC",
+            "I2C",
+            "I2CSLAVE",
+            "INTERRUPTIN",
+            "PORTIN",
+            "PORTINOUT",
+            "PORTOUT",
+            "PWMOUT",
+            "RESET_REASON",
+            "SERIAL",
+            "SERIAL_FC",
+            "SERIAL_ASYNCH",
+            "SLEEP",
+            "SPI",
+            "SPI_ASYNCH",
+            "SPISLAVE",
+            "STDIO_MESSAGES",
+            "TRNG",
+            "FLASH",
+            "USBDEVICE",
+            "WATCHDOG"
+        ],
+        "release_versions": [
+            "2",
+            "5"
+        ],
+        "device_name": "MK64FN1M0xxx12",
+        "bootloader_supported": true,
+        "overrides": {
+            "network-default-interface-type": "ETHERNET"
+        },
+        "supported_c_libs": {
+            "arm": [
+                "std",
+                "small"
+            ],
+            "gcc_arm": [
+                "std",
+                "small"
+            ],
+            "iar": [
+                "std"
+            ]
+        }
+    }
+```
+
+Note the config options `macros_add`, `extra_labels_add` and `device_has_add`. To add or remove `macros`, `extra_labels` or target capabilities, a PSA target definition **must** use [macros/extra_labels/device_has]`_add` or [macros/extra_labels/device_has]`_remove` (not `macros`, `extra_labels` or `device_has`).
+
```

Comment:
You moved the links from macros , extra_labels , device_has  and Features  to a new section. Wouldn’t it be easier for the user, if those would have been hyperlinks rather than links in a different section?

Reply:
But we had it three or four times in the docs. That can make people try to figure out whether there's a difference between those times, which is a waste of effort, or - worse - can make them skim the end of each section on the assumption that more of it repeats.

Reply:
Irit: But we had it three or four times in the docs. That can make people try ti figure out whether there's a different between those times, which is a waste of effort, or - worse - can make them skim the end of each section on the assumption that more of it repeats

Reply:
@iriark01 Agreed, that makes sense.